src/types.ts

```diff
@@ -88,6 +90,61 @@ export interface ICourierMessageGetResponse {
   status: string;
 }
 
+interface ICourierBrandSettings {
+  colors?: {
+    primary: string;
+    secondary: string;
+    tertiary: string;
+  };
+  email?: {
+    footer: object;
+    header: object;
+  };
+}
+
+interface ICourierBrandSnippets {
+  items: Array<{
+    format: string;
+    name: string;
+    value: string;
+  }>;
+}
+
+export interface ICourierBrand {
+  created: number;
+  id?: string;
+  name: string;
+  published: number;
+  settings: ICourierBrandSettings;
+  updated: number;
+  snippets?: ICourierBrandSnippets;
+  version: string;
+}
+
+export interface ICourierPaging {
+  cursor?: string;
+  more: boolean;
+}
+
+export interface ICourierBrandPostParameters {
+  id?: string;
+  name: string;
+  settings: ICourierBrandSettings;
+  snippets?: ICourierBrandSnippets;
+}
+
+export interface ICourierBrandPutParameters {
+  id: string;
+  name: string;
+  settings: ICourierBrandSettings;
+  snippets?: ICourierBrandSnippets;
+}
```

Comment:
Could consolidate into one interface or introduce a base interface that these two can extend.

Reply:
I figured we could do something like this. I wasn't sure how to make the `id` required only for the `PUT`.